### DISCUSSION TITLE
Rainbow the calendar in sidebar/add-on in Gmail

### DISCUSSION THREAD

**Discussion by mpinsky6** [coedit/gcal-rainbow-surf#3] [archive]
Hey co-team!

I love the rainbow surf, it is so cool for all of the shared calendars (especially with my family lol) - it doesn't work on the side bar. When I am in gmail, sometimes I am working with my calendar open in a minimal view on the right side bar.  The events are still duplicated on that. Thoughts?

Thanks, mpinsky6

**Comment by ezdub** [comment]
@mpinsky6 Great suggestion!  
We'll look into making the rainbows appear inside the Gmail sidebar (aka Add-on) for Google Calendar.

**Reply by ezdub** [reply]
Hi, I wanted to follow up that modifying the sidebar in Gmail would involve additional permissions for this extension to inject code in gmail.com site. We would want this to be optionally available since it may sound risky to some people & since I suspect not everyone needs it we'll wait for more demand.

**Everyone interested in seeing merged cal events in the gmail sidebar, please thumbs-up this post.**
